### PR TITLE
ENH: DiSCo

### PR DESCRIPTION
**Pull request by AntoineTheb**
PM asked me to present the challenge, here it is.

### CODE REVIEW

File: article/_posts/2021-07-29-DiSCo.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+layout: review
+title: "The Diffusion-Simulated Connectivity (DiSCo) Challenge"
+tags: tractography tractometry diffusion
```
Comment (jhlegarreta) [inline article/_posts/2021-07-29-DiSCo.md:4]: Maybe "diffusion-mri" is a more descriptive/complete tag?

File: article/_posts/2021-07-29-DiSCo.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+layout: review
```
Comment (jhlegarreta) [inline article/_posts/2021-07-29-DiSCo.md:2]: Maybe the "blog" layout fits better for this entry? Otherwise, some header fields specific to the "review" type are missing, and thus I guess the site would not be built correctly (?).

File: article/_posts/2021-07-29-DiSCo.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+title: "The Diffusion-Simulated Connectivity (DiSCo) Challenge"
+tags: tractography tractometry diffusion
+author: "Antoine Théberge"
+pdf: "http://hardi.epfl.ch/static/events/2021_challenge/"
+---
+
+# Introduction
+The Diffusion-Simulated Connectivity (DiSCo) Challenge is a connectivity estimation challenge presented at CDMRI, a workshop organized at MICCAI 2021. The goal of the challenge is to estimate the connectivity of a synthetic phantom composed of 16 regions of interests connected by interlacing fiber bundles. The participating teams must provide their own connectivity matrix and the team with the highest correlation coefficient with the ground-truth matrix will be declared winner.
```
Comment (jhlegarreta) [inline article/_posts/2021-07-29-DiSCo.md:10]: "regions of interests": "regions of interest"

File: article/_posts/2021-07-29-DiSCo.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+# Motivations
+
+The estimation of brain structural connectivity from diffusion-weighted magnetic resonance (DW-MR) images  is still an open subject and depends on many steps. Roughly, connectivity estimation from a set of DW-MR images acquisitions goes as follow
```
Comment (jhlegarreta) [inline article/_posts/2021-07-29-DiSCo.md:14]: "as follow": "as follows"

File: article/_posts/2021-07-29-DiSCo.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+Because each step has its own set of methods and each methods has its own set of parameters, it is primordial to evaluate the performance of the pipelines currently used.
+
+# Data
+
+![](/article/images/disco/fig1.jpg)
+
+> Left: ROIs. Middle: Strands. Right: Connectivity matrix.
+
+To do so, the organizing team has released three versions (training, validation and testing) of the same dataset, each with the same regions of interests (ROIs) but different connectivity matrices. Each dataset was generated by deforming large tubular fibers to form arcs between ROIs, with energy parameters controlling the length, curvature and overlap with others. Each strand was then separated into smaller virtual axons of varying diameter.
```
Comment (jhlegarreta) [inline article/_posts/2021-07-29-DiSCo.md:30]: "regions of interests": "regions of interest"

Comment (jhlegarreta) [inline article/_posts/2021-07-29-DiSCo.md:30]: Rather than the "same dataset", if connectivity matrices are different, wouldn't they be different datasets with the same underlying model to generate the data and the same regions of interest?

File: article/_posts/2021-07-29-DiSCo.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+The virtual axons were then used to compute intra and extra axonal water compartments, which were then used to generate a virtual DW-MR signal.
+
+The training dataset provides the ground-truth connectivity matrix, a map of the ROIs, the noiseless DWI, a noisy version of the DWI (SNR~30), the "white-matter" mask, the actual virtual strands and more. The validation datasets provides the noisy (SNR~30) DWI, a map of the ROIs and the ground-truth connectivity matrix, and the testing dataset provides the noisy DWI and a map of the ROIs.
+
+A script to compute the correlation coefficient between two matrices has also be provided.
```
Comment (jhlegarreta) [inline article/_posts/2021-07-29-DiSCo.md:36]: "be provided": "been provided"

File: article/_posts/2021-07-29-DiSCo.md (hunk 1 above)
Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md]: Changed for dMRI in d7c2ce9eb767b433be27444ba37507eda745dfad

File: article/_posts/2021-07-29-DiSCo.md (hunk 3 above)
Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md]: Indeed, fixed in d6aefbc0de10be4f97613718234cedae4bd20da6 and added acronym

File: article/_posts/2021-07-29-DiSCo.md (hunk 2 above)
Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md]: Changed in d6aefbc0de10be4f97613718234cedae4bd20da6

File: article/_posts/2021-07-29-DiSCo.md (hunk 4 above)
Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md]: Fixed in d6aefbc0de10be4f97613718234cedae4bd20da6

File: article/_posts/2021-07-29-DiSCo.md (hunk 5 above)
Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md]: Replaced for the acronym in d6aefbc0de10be4f97613718234cedae4bd20da6

Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md]: Reworded in d6aefbc0de10be4f97613718234cedae4bd20da6

File: article/_posts/2021-07-29-DiSCo.md (hunk 6 above)
Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md]: Fixed in d6aefbc0de10be4f97613718234cedae4bd20da6

File: article/_posts/2021-07-29-DiSCo.md (hunk 5 above)
Comment (jhlegarreta) [reply article/_posts/2021-07-29-DiSCo.md]: Thanks. I later realized that maybe it was indeed the same dataset, and using that data they might have explicitly disallowed some regions to be connected to generate each subset. Sorry if it was me that misunderstood the principle.

**Review by jhlegarreta — APPROVED** [review]

File: article/_posts/2021-07-29-DiSCo.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+- Signal processing and modelling
+- Tractography
+- Tractogram filtering and post-processing
+- Connectivity estimation
+
+Because each step has its own set of methods and each methods has its own set of parameters, it is primordial to evaluate the performance of the pipelines currently used.
+
+# Data
+
+![](/article/images/disco/fig1.jpg)
```
Comment (ThierryJudge) [inline article/_posts/2021-07-29-DiSCo.md:26]: This image doest not seem to be included in the pull request

**Review by ThierryJudge — CHANGES_REQUESTED** [review]

File: article/_posts/2021-07-29-DiSCo.md (hunk 7 above)
Comment (AntoineTheb) [reply article/_posts/2021-07-29-DiSCo.md:26]: Woops indeed, fixed in c50a3c81507212f26a998f2e07a298c29ea7d25e

**Review by ThierryJudge — APPROVED** [review]
Looks good
